client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -63,6 +65,23 @@ function ReaderSubscriptionListItem( {
 		return <ReaderSubscriptionListItemPlaceholder />;
 	}
 
+	function recordEvent( name ) {
```

Comment:
scratch this line of questions.  remembering now that none of this matters for a leaf-component like this where it won't trigger rerenders in a way that matters